packages/documentation/copy/en/reference/Security.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Security"
+layout: docs
+permalink: /docs/handbook/security
+oneline: "Federated Execution with Security"
+preamble: >
+---
+
+The current implementation of federated execution lacks authentication of federates from joining the federation. Also communication security is not implemented to prevent adversarial federates sending malicious messages (e.g.. bad tags or bad sensor readings) to the RTI or other federates.
```

Comment:
```suggestion
By default, there is no secure authentication happening when a federate joins a federation, and data exchanged by federates is not encrypted. The `auth` target property can be used to change this.
```

Reply:
Commited all your changes at once including this and below.